src/share/util/shr_dmodel_mod.F90

```diff
@@ -771,7 +769,7 @@ subroutine shr_dmodel_readstrm(stream, pio_subsystem, pio_iotype, pio_iodesc, gs
 
   call t_stopf(trim(lstr)//'_setup')
 
-  if (pio_iotype == iotype_std_netcdf) then
+  if (pio_iotype == pio_iotype_netcdf) then
```

Comment:
I'm afraid I gave you bad information, by setting this to -99 the logic was that if the PIO_TYPENAME was not explicitly set for this component bypass pio and use netcdf directly.   But in recent versions we always set the pio_typename and so this code is never exercised.   The change you've made will cause pio to be bypassed here if you set pio_typename='netcdf' I dont think that we want that.